brave/browser-laptop
# allow browser-laptop to join an existing sync profile

by entering the 16 code words

Test Plan:

prep:
1. add `"start2": "node ./tools/start.js --user-data-dir=brave-development-2 --debug=5859 --enable-logging --v=0 --enable-extension-activity-logging --enable-sandbox-logging --enable-dcheck"` to package.json
2. change appConfig fetchInterval to 5000
2. remove brave-development-2 if it exists

play:
1. enable sync in pyramid 1
2. in pyramid 2, go to about:preferences#sync and choose 'i have an existing sync code'
3. in pyramid 1, click on 'add a new device' and then click 'show secret code words'
4. copy the code words from pyramid 1 into the text box in pyramid 2. click 'set up sync' and restart
5. bookmark any site in pyramid 1. it should appear bookmarked in pyramid 2.
